x/ccv/provider/client/cli/query.go

```diff
@@ -19,6 +19,9 @@ func NewQueryCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(NewQueryConsumerGenesisCmd())
+	cmd.AddCommand(CmdConsumerChains())
```

Comment:
Could we make the naming more consistent here? I.e. almost all of them start with Cmd but one of them begins with Cmd.

Reply:
`Done.`